Crash on Manage Note Types

###### Reproduction Steps

1. open current alpha
2. from deck picker, click top-right 3-dot menu, select "manage note types"
3. pick any note type


###### Expected Result

You go to the type editor


###### Actual Result

crash


###### Debug info

09-17 01:25:21.379 28777-28777/com.ichi2.anki E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.ichi2.anki, PID: 28777
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.ModelFieldEditor}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ListView.setAdapter(android.widget.ListAdapter)' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2646)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6077)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ListView.setAdapter(android.widget.ListAdapter)' on a null object reference
        at com.ichi2.anki.ModelFieldEditor.createfieldLabels(ModelFieldEditor.java:140)
        at com.ichi2.anki.ModelFieldEditor.onCollectionLoaded(ModelFieldEditor.java:125)
        at com.ichi2.anki.AnkiActivity.startLoadingCollection(AnkiActivity.java:255)
        at com.ichi2.anki.ModelFieldEditor.onCreate(ModelFieldEditor.java:81)
        at android.app.Activity.performCreate(Activity.java:6662)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)


###### Research
It looks like someone runs a big automated UI test using roboelectric/espresso on our alphas, which is kind of neat - I saw it there from the prelaunch report
https://play.google.com/apps/publish/?account=5338425030304417978#PreLaunchReportPlace:p=com.ichi2.anki&plrvc=20900133&plrtab=CRASH

--- a/AnkiDroid/src/main/java/com/ichi2/anki/ModelFieldEditor.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/ModelFieldEditor.java
@@ -78,7 +78,6 @@ public class ModelFieldEditor extends AnkiActivity {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.model_field_editor);
-        startLoadingCollection();
 
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         mFieldLabelView = (ListView) findViewById(R.id.note_type_editor_fields);
@@ -87,11 +86,11 @@ public class ModelFieldEditor extends AnkiActivity {
             setSupportActionBar(toolbar);
         }
 
-
         if (getSupportActionBar() != null) {
             getSupportActionBar().setTitle(R.string.model_field_editor_title);
             getSupportActionBar().setSubtitle(getIntent().getStringExtra("title"));
         }
+        startLoadingCollection();
     }
 
 
@@ -120,6 +119,7 @@ public class ModelFieldEditor extends AnkiActivity {
 
     @Override
     protected void onCollectionLoaded(Collection col) {
+        super.onCollectionLoaded(col);
         this.mCol = col;
         setupLabels();
         createfieldLabels();

--- a/AnkiDroid/src/main/java/com/ichi2/anki/Statistics.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/Statistics.java
@@ -91,6 +91,7 @@ public class Statistics extends NavigationDrawerActivity implements DeckDropDown
     @Override
     protected void onCollectionLoaded(Collection col) {
         Timber.d("onCollectionLoaded()");
+        super.onCollectionLoaded(col);
         SlidingTabLayout slidingTabLayout;
         // Add drop-down menu to select deck to action bar.
         mDropDownDecks = getCol().getDecks().allSorted();

